### PR TITLE
Updated Animations library, updated sample and added Animations nuspec

### PR DESCRIPTION
**Pull request by jsuarezruiz**
Added ColorAnimation to the Animations Library
Updated Animations Sample
Added Animations nuspec

### CODE REVIEW

File: nuget/Animations.nuspec [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+       <title>Xamarin.Forms Community Toolkit: Animations</title>
+       <authors>FormsCommunityToolkit</authors>
+       <owners>FormsCommunityToolkit</owners>
+       <projectUrl>https://github.com/FormsCommunityToolkit/FormsCommunityToolkit</projectUrl>
+       <licenseUrl>https://github.com/FormsCommunityToolkit/FormsCommunityToolkit/blob/master/LICENSE</licenseUrl>
+       <iconUrl>https://github.com/FormsCommunityToolkit/FormsCommunityToolkit/blob/master/Media/OrganonLogo.64x64.png</iconUrl>
+       <requireLicenseAcceptance>false</requireLicenseAcceptance>
+       <description>
+         Xamarin.Forms Community Toolkit Animations Library
+         Built against: 2.3.2.127
```
Comment (bartlannoeye) [inline nuget/Animations.nuspec]: I see another version in the samples?

File: src/Animations/Animations/Animations/ColorAnimation.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
+﻿using FormsCommunityToolkit.Animations.Extensions;
+using System;
```
Comment (bartlannoeye) [inline src/Animations/Animations/Animations/ColorAnimation.cs]: system usings first

File: nuget/Animations.nuspec (hunk 1 above)
Comment (jsuarezruiz) [reply nuget/Animations.nuspec]: Yes, you are right, the 2.3.4.247 version.

File: src/Animations/Animations/Animations/ColorAnimation.cs (hunk 2 above)
Comment (jsuarezruiz) [reply src/Animations/Animations/Animations/ColorAnimation.cs]: Ok!

**Pull request merged**